decrypter.py

```diff
@@ -0,0 +1,74 @@
+from cryptography.hazmat.backends.openssl.backend import backend
+from cryptography.hazmat.primitives import serialization
+from cryptography.hazmat.primitives.ciphers import Cipher, algorithms, modes
+from cryptography.hazmat.primitives import hashes
+from cryptography.hazmat.primitives.asymmetric import padding
+from cryptography.exceptions import InvalidTag, InternalError
+
+import settings
+import base64
+import jwt
+import logging
+import json
+
+IV_EXPECTED_LENGTH = 12
+CEK_EXPECT_LENGTH = 32
+
+class Decrypter(object):
+    def __init__(self):
+        self.public_key = serialization.load_pem_public_key(
+            settings.PUBLIC_KEY.encode(),
+            backend=backend
+        )
+        self.private_key = serialization.load_pem_private_key(
+            settings.PRIVATE_SIGNING_KEY.encode(),
+            password=self._to_bytes(settings.PRIVATE_SIGNING_KEY_PASSWORD),
+            backend=backend
+        )
+
+    def _to_bytes(self, bytes_or_str):
+        if isinstance(bytes_or_str, str):
+            value = bytes_or_str.encode()
+        else:
+            value = bytes_or_str
+        return value
+
```

Comment:
We probably should check the KID value in the header (sorry that's not in my decrypt code)